src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -443,7 +443,8 @@ object ManagedIndexRunner : ScheduledJobRunner,
             actionMetaData = null,
             stepMetaData = null,
             policyRetryInfo = PolicyRetryInfoMetaData(failed = true, consumedRetries = 0),
-            info = mapOf("message" to "Fail to load policy: $policyID")
+            info = mapOf("message" to "Fail to load policy: $policyID"),
+            enabled = true
```

Comment:
What was this `enabled = true` change for?

Reply:
For Kibana side, previously kibana query config index directly for managed-index and get this enabled field

now we can only provide this information through explain API response to Kibana, so add this field.

Reply:
What happens when someone disables the job and this wasn't updated and now metadata says enabled=true and job says enabled=false? You're going to run into a lot of issues trying to keep this in sync with source.

Reply:
As discussed offline, let's not introduce another enabled flag here as keeping it in sync with actual enabled flag is going to introduce issues. Instead just query the config index for the job and get the actual enabled flag value from the source when a user calls the explain API.